demos/place_recognition_demo/python/place_recognition_demo/place_recognition.py

```diff
@@ -31,24 +31,24 @@ class IEModel: # pylint: disable=too-few-public-methods
     def __init__(self, model_path, device, cpu_extension):
         log.info('OpenVINO Inference Engine')
         log.info('\tbuild: {}'.format(get_version()))
-        ie = IECore()
+        core = Core()
         if cpu_extension and device == 'CPU':
-            ie.add_extension(cpu_extension, 'CPU')
+            core.add_extension(cpu_extension, 'CPU')
 
         log.info('Reading model {}'.format(model_path))
-        self.net = ie.read_network(model_path, model_path.with_suffix('.bin'))
-        self.input_name = next(iter(self.net.input_info))
-        self.output_name = next(iter(self.net.outputs))
-        self.input_size = self.net.input_info[self.input_name].input_data.shape
-        self.exec_net = ie.load_network(network=self.net, device_name=device)
+        self.model = core.read_model(model_path, model_path.with_suffix('.bin'))
+        self.input_tensor_name = self.model.inputs[0].get_any_name()
+        self.input_size = list(self.model.input(self.input_tensor_name).shape)
```

Comment:
is it necessary to wrap shape to list?

Reply:
We cannot apply slicing to tensor shape as in the old API, so the solution here is to wrap to list. Replaced by getting the width and height by indices.